.circleci/config.yml

```
@@ -100,67 +104,16 @@ jobs:
       shell: bash --login -eo pipefail
     <<: *test-base
 
-  test-lambdas-indexer:
-    docker:
-      - image: circleci/python:3.6-jessie
-    steps:
-      - checkout
-      - run:
-          name: Setup
-          command: |
-            virtualenv venv
-            . venv/bin/activate
-            pip install lambdas/shared/
-            pip install -r lambdas/es/indexer/requirements.txt
-            pip install -r lambdas/es/indexer/test-requirements.txt
-      - run:
-          name: Pytest
-          command: |
-            . venv/bin/activate
-            cd lambdas/es/indexer && pytest --cov=./
-
-  test-lambdas-preview:
-    docker:
-      - image: circleci/python:3.6-jessie
-    steps:
-      - checkout
-      - run:
-          name: Setup
-          command: |
-            virtualenv venv
-            . venv/bin/activate
-            pip install lambdas/shared/
-            pip install -r lambdas/preview/requirements.txt
-            pip install -r lambdas/preview/test-requirements.txt
-      - run:
-          name: Pytest
-          command: |
-            . venv/bin/activate
-            cd lambdas/preview && pytest --cov=./
-
-  test-lambdas-s3select:
-    docker:
-      - image: circleci/python:3.6-jessie
-    steps:
-      - checkout
-      - run:
-          name: Setup
-          command: |
-            virtualenv venv
-            . venv/bin/activate
-            pip install lambdas/shared/
-            pip install lambdas/s3select
-            pip install -r lambdas/s3select/requirements.txt
-            pip install -r lambdas/s3select/test-requirements.txt
-      - run:
-          name: Pytest
-          command: |
-            . venv/bin/activate
-            cd lambdas/s3select && pytest --cov=./
-
-  test-lambdas-search:
+  test-lambda:
     docker:
       - image: circleci/python:3.6-jessie
+    description: "Test lambdas"
+    environment:
+      QUILT_DISABLE_USAGE_METRICS: true
+    parameters:
+      path:
+        description: "lambdas/$path"
```

Sorry, here's a better description:
```suggestion
        description: "Relative path to lambda root including setup.py, e.g. 'es/indexer' for lambdas/es/indexer"
```